serverless.yml

```diff
@@ -58,3 +57,12 @@ functions:
       - http:
           path: dynamodb-example
           method: get
+  redirectUrl:
+    handler: bin/redirectUrl
+    events:
+      - http:
+          path: redirect-url
+          method: get
```

Comment:
hmm, it would sound better if we could add `/oauth/` somewhere in the URL. Also I have seen URLs like `/oauth/callback` in many apps. The redirect-url refers to the param on the Oauth consent screen that redirects you to a "callback" page/function

I would suggest calling this function oauthCallback (and also rename the package accordingly - it seems like all lowercase is the norm in go so just `oauthcallback` for the package name ?)

Reply:
I named it redirect-url because Slack calls it redirect URL in the configuration:

<img width="706" alt="Screenshot 2020-03-08 at 15 24 09" src="https://user-images.githubusercontent.com/7913565/76157768-e24a0080-6150-11ea-98c9-8b697194c82c.png">